lib/resourceapply/apps.go

```diff
@@ -34,12 +35,12 @@ func ApplyDeployment(client appsclientv1.DeploymentsGetter, required *appsv1.Dep
 	existing, err := client.Deployments(required.Namespace).Get(required.Name, metav1.GetOptions{})
 	if apierrors.IsNotFound(err) {
 		actual, err := client.Deployments(required.Namespace).Create(required)
+		actualGet, err := client.Deployments(required.Namespace).Get(required.Name, metav1.GetOptions{})
```

Comment:
why we need this?

Reply:
added for debugging. forgot to remove. will remove thanks!